pkg/util/util.go

```diff
@@ -1512,6 +1513,26 @@ func IsStringArrayUnorderedEqual(stringsArrayA, stringsArrayB []string) bool {
 	return true
 }
 
+// EnsureNoobaaAnnotations ensures that the resource has all mandatory meta fields
+func EnsureNoobaaAnnotations(object metav1.Object) bool {
+	updated := false
+
+	annotations := object.GetAnnotations()
+	if annotations == nil {
+		object.SetAnnotations(map[string]string{
+			DisableDefaultTopologyConstraints: "false",
```

Comment:
@guymguym I didn't get what you meant by `leave this out.`, I added this and set it to default `true` only because the user should get a chance to know this configuration we have in Noobaa as you suggested.

Reply:
no need to set it on the CR though, it should be documented instead.